matrix_synapse_saml_mozilla/res/style.css

```diff
@@ -126,7 +126,8 @@ body {
   transition: background-color .1s ease-in-out;
   -webkit-appearance: none;
   -moz-appearance: none;
-  appearance: none; }
+  appearance: none;
+  cursor: pointer; }
```

Comment:
```suggestion
  cursor: pointer;
}
```

Reply:
You would think so, but all the other code in the file does this, and apparently it is *a thing*.

Reply:
wtf
how is it a thing?
anyway w/e